V1.1 #3

sabaibamosu opened the pull request:
v1.1-#3の追加

sabaibamosu committed:
v1.1-#3

v1.1-#3の追加

sabaibamosu committed:
v1.1-#3

v1.1-#3の追加

sabaibamosu committed:
v1.1-#3

v1.1-#3の追加

sabaibamosu committed:
v1.1-#3

v1.1-#3の追加

sabaibamosu committed:
v1.1-#3

v1.1-#3のバグを修正
(GUIのアイテムを取り出すことができないように)

sabaibamosu committed:
v1.1-#3

Item変数へのアクセス方法を変更

sabaibamosu committed:
v1.1-#3

企画書に準拠したデザインに変更

sabaibamosu committed:
v1.1-#3

変数及びアイテムにおける表記揺れの修正
(入手した頭が消えるバグを修正)

RunqRun commented on Gacha.sk:
```diff
@@ -0,0 +1,98 @@
+            set {_n} to {_n} * 100
+            profileExpIncrease({_player}, {_n}, "GACHA_USE")
+    set {_name} to getDefine("GACHA_PRIZE_ICON::%{_itemID}%")
+    send "&f&l鉄ガチャ &8» %name of {_name}% &aを引き当てた。" to {_player} if {_gachaID} is "0"
```
後からの変更のために、OptionsまたはDefine.skで定義しましょう。
Optionsの場合は、パース（≒コンパイル）時にそのまま置換されるため、不確定な部分はそのままname of {_name}で動作します。
Define.skで定義するときは不確定な部分を${}にしてsubstitution関数を活用しましょう。

sabaibamosu committed:
v1.1-#3

修正指示に対応

sabaibamosu committed:
fix: 表記修正&跳躍力修正

EXP → GOLD
GOLD → EXP
ウィーザースケルトン → ウィザースケルトン
EXP/GOLD付与 → ●●EXP/GOLD付与

跳躍力Lv1 → Lv2

sabaibamosu committed:
文法エラー修正

ダブルクオーテーションをつけ忘れていた

sabaibamosu committed:
fix: 表記揺れ&論理エラー修正

ウィーザースケルトン → ウィザースケルトン
number(小数点) → integer(整数)

sabaibamosu committed:
エラー,表記,NPC(スキン)の修正

GACHA_T → 村人スキンをtypeより設定
(それに伴い、不要なGACHA_T.pngを消去)
水入り瓶 → 「効果なし」表示の消去
エラー → 論理&文法エラー修正
